docs/getting_started.md

```diff
@@ -287,27 +272,27 @@ for i in range(100):
 '...and now we\'re done!'
 ```
 
-## Record a screencast
+## Part 4: Recording a screencast (optional)
 
-After you've built a Streamlit app, you may want to discuss some of it with co-workers over email or Slack, or share it with the world on Twitter. A great way to do that is with Streamlit's built-in screencast recorder. With it, you can record, narrate, stop, save, and share with a few clicks.
+After you've built a Streamlit app, you may want to discuss some of it with co-workers over email or Slack, or share it with the world on Twitter.
+A great way to share your app is by using the Streamlit built-in screencast recorder. With it, you can record, narrate, stop, save, and share with a few clicks.
 
-To start a screencast, locate the menu in the upper right corner of your app (**☰**), select **Record a screencast**, and follow the prompts. Before the recording starts, you'll see a countdown — this means it's showtime.
+To start a screencast, locate the menu in the upper right corner of your app (**☰**), select **Record a screencast**, and follow the prompts.
+Before the recording starts, you'll see a countdown — this means it's showtime.
 
 To stop your screencast, go back to the menu (**☰**) and select **Stop recording** (or hit the **ESC** key). Follow the prompts to preview your recording and save it to disk. That's it, you're ready to share your Streamlit app.
 
 ```eval_rst
 .. image:: ./media/screenshare.gif
 ```
 
-## Get help
+## Getting help with Streamlit
 
-That's it for getting started, now you can go and build your own apps! If you
-run into difficulties here are a few things you can do.
+If you run into difficulties building your own Streamlit apps, here are a few resources you can turn to:
 
-- Check out our [community forum](https://discuss.streamlit.io/) and post a
-  question
-- Quick help from command line with `$ streamlit --help`
-- Read more documentation! Check out:
+- Stop by the [community forum](https://discuss.streamlit.io/) and post a question
+- Get quick help from command line with `$ streamlit --help`
```

Comment:
from *the* command line

Although I'd love it if we could shout instructions at Command Line like we do with Siri :D

Reply:
:+1: